common/sgx/tcbinfo.c

```diff
@@ -590,16 +659,59 @@ static oe_result_t _read_tcb_info(
         }
     }
 
-    if (parsed_info->version == 2)
+    if (parsed_info->version == 3)
```

Comment:
This block is 95% identical to the version 2 block. Can we consider merging the 2 blocks and leave the diffs in _read_tcb_info_tcb_level_v*()? My concern is that as more versions come in the future, we will have to maintain multiple copies of similar code.